scss/components/_banner.scss

```diff
@@ -1,6 +1,18 @@
 .banner {
-    background-color: $astral;
-    color: $white;
+    background-color: $white;
+
+    &--prototype {
+        background-color: $carrot;
+        color: $white;
+    }
+
+    &__wrapper {
```

Comment:
Isn't this a modifier? E.g. it adds a border to the banner `banner--bottom-border`

If not, can we call it something more descriptive than wrapper?